streams/src/test/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilderTest.java

```diff
@@ -528,13 +522,12 @@ public void shouldAddInternalTopicConfigForWindowStores() throws Exception {
         assertTrue(topicConfig instanceof WindowedChangelogTopicConfig);
     }
 
-    @SuppressWarnings("unchecked")
     @Test
-    public void shouldAddInternalTopicConfigForNonWindowStores() throws Exception {
+    public void shouldAddInternalTopicConfigForNonWindowStores() {
         builder.setApplicationId("appId");
         builder.addSource(null, "source", null, null, null, "topic");
         builder.addProcessor("processor", new MockProcessorSupplier(), "source");
-        builder.addStateStore(new MockStateStoreSupplier("store", true), "processor");
+        builder.addStateStore(storeBuilder, "processor");
```

Comment:
the original store is persistent -- but `storeBuilder` is in-memory -- change intended?

Reply:
For this test it does not matter. I was just reusing the store instance.